addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -168,7 +168,10 @@
             <t t-set="urlPlaceholder">https://www.odoo.com/logo.png</t>
             <t t-set="addText">Add URL</t>
             <t t-set="searchPlaceholder">Search an image</t>
-            <t t-set="urlWarningTitle">The URL does not contain any image. The file will be added in the document section.</t>
+            <t t-set="urlWarningTitle">
+                Image's format uploaded not supported.
+                Try with: <t t-esc="widget.IMAGE_EXTENSIONS.join(', ')"/>
```

Comment:
I would avoid t-esc inside translated sentences but I guess it is fine when at the end after two points? Feels fragile though, I can easily see someone innocently adding "for example" at the end of the sentence.

The sentence also need to be trimmed otherwise, the many spaces are shown in the tooltip (at least on FF?)

Reply:
Fixed both